contracts/optimistic-ethereum/OVM/execution/OVM_StateManager.sol

```diff
@@ -10,6 +10,14 @@ import { iOVM_StateManager } from "../../iOVM/execution/iOVM_StateManager.sol";
 
 /**
  * @title OVM_StateManager
+ * @dev The State Manager contract holds all storage values for contracts in the OVM. It can only be written to by the 
+ * the Execution Manager and State Transitioner. It runs on L1 during the setup and execution of a fraud proof.
```

Comment:
Technically it is also used in the `POST_EXECUTION` phase; we read what OVM storage has been updated to generate the `postStateRoot`.